app/src/main/AndroidManifest.xml

```diff
@@ -13,7 +21,17 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
-        android:theme="@android:style/Theme.DeviceDefault">
+        android:name=".HeartRateApplication"
+        android:theme="@style/AppTheme">
+        <receiver
```

Comment:
After moving HR manager and receiver to data layer please move this too into manifest for the data layer

Reply:
done